### ISSUE TITLE
Unexpected column names in query result when using joins

### ISSUE THREAD

**Issue by AnEnigmaticBug** [tekartik/sqflite#229]
I have the following query:
```
SELECT a.id, a.name, b.id
   FROM apple a
  INNER JOIN banana b ON <Condition>
```
The resulting maps(obtained by using `rawQuery`) have the following keys: `id`, `name`, `id`

This is unexpected for two reasons:

- Maps shouldn't have duplicate keys
- Since I used `a.id` in the query, I expected the corresponding key to be `a.id`

I got around this problem by renaming the conflicting columns using `AS`. But this may bite people unexpectedly.

**Comment by alextekartik** [comment]
Thanks for the report. Indeed the keys should not be duplicated. I'll fix that. Since sqflite does not do any sql parsing, the keys are whatever the native side says and on both iOS and Android `a.id` will be reported as `id` and the solution is to use `AS` as you did.

**Comment by AnEnigmaticBug** [comment]
Just a request: if you do make the keys `a.id` and `b.id`, please document this on your dartpub's README section. Maybe just mention how the key names are chosen.

Thanks for your quick reply. I really appreciate the work you've put in your packages.

**Comment by alextekartik** [comment]
There will be no breaking changes, besides fixing the duplicated keys. Currently in the fake map, the last column wins and that will remain the same, and `a.id` and `b.id` are still reported as `id`.

See this test: https://github.com/tekartik/sqflite/blob/316b9f649f6a8bb4d6f754b6246380743534a1b2/sqflite/example/test_driver/sqflite_test.dart#L115-L119

I published a version 1.1.6-dev.1 with the changes if you want to try.
